packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -32,6 +32,8 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
   private _stickyContentBottom = React.createRef<HTMLDivElement>();
   private _nonStickyContent = React.createRef<HTMLDivElement>();
   private _placeHolder = React.createRef<HTMLDivElement>();
+  private _isFocusActive: boolean;
```

Comment:
I'm not too clear on what focus is active on based on this naming. Is focus active on a sticky component?

Reply:
Good point, that could be more descriptive. Looking at it more, we might not actually even need the `_isFocusActive` boolean. We could just set `_activeElement` to undefined where we set `_isFocusActive` to false. I'll push those changes.